### PR TITLE
Convert examples cXY to test cases Pt. 4

### PR DESCRIPTION
**Pull request by Nicoretti**
- **Move edge_cases and their  ddl to integration level fixtures**
- **Add tests for edge cases**
- **Remove unused imports**
- **WIP: Tls tests**

### CODE REVIEW

File: test/integration/edge_cases_test.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    connection.commit()
+
+    yield table
+
+
+@pytest.mark.edge_case
+def test_insert(connection, table, edge_cases):
+    stmt = (
+        "INSERT INTO edge_case VALUES ({DEC36_0!d}, {DEC36_36!d}, {DBL!f}, "
```
[the remark this answers is not in the document]
Comment (Nicoretti) [reply test/integration/edge_cases_test.py:34]: Good catch, thx!

File: test/integration/json_test.py [hunk 2]
```diff
@@ -128,9 +72,27 @@ def test_select(table, connection_factory, edge_cases, json_lib):
 
     expected = {
         # Biggest values
-        ("9" * 36, f"0.{'9' * 36}", 1.7e308, True, "9999-12-31", "9999-12-31 23:59:59.999000", "ひ" * 100, 2000000),
+        (
```
[the remark this answers is not in the document]
Comment (Nicoretti) [reply test/integration/json_test.py:75]: Initially wanted to do that but there are a few minor type differences.

**Review by ahsimb — APPROVED** [review]
